spec/controllers/password_resets_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper' 
+
+describe PasswordResetsController do 
+
+  describe "GET show" do 
+
+    before do 
+      @alice = Fabricate(:user, token: '12345')
```

Comment:
You should use `let` rather than instance variables. Read more [here](http://betterspecs.org/#let).